main/lib/ttree/src/TTreeEventConverter.cc

```diff
@@ -69,7 +69,9 @@ namespace eudaq{
     uint32_t id = d1->GetType();
     auto cvt = Factory<TTreeEventConverter>::MakeUnique(id);
     if(cvt){
-      return cvt->Converting(d1, d2, conf);
+      cvt->Converting(d1, d2, conf);
+      d2->Fill();
+      return true;
```

Comment:
You might possibly the first user of the TTree converter.  
I'm not sure if the Fill() at line  67 and line 73 have race condition or not.
Do you have any test run with RootMonitor? Which kind of Event Data is feed into the TTreeConverter/RootMonitor?

Reply:
Indeed, this is what I understood from a previous conversation with @lhuth :-)
As told below, I will come up with a proper example after discussion with my colleagues on what may be released "to the world" from our previous test beam runs / lab fiddling.